### DISCUSSION TITLE
LED Macros

### DISCUSSION THREAD

**Discussion by cmenard** [xblax/flashforge_ad5m_klipper_mod#130] [General]
I added LEDs to my AD5M and wanted a way to turn them on and off.  Here's the macros I came up with.

`[gcode_macro LED_ON]
gcode:
  {% set Brightness = params.VALUE|default(80)|float %}
  SET_LED LED=chamber_led WHITE={Brightness / 100} SYNC=0 TRANSMIT=1

[gcode_macro LED_OFF]
gcode:
 LED_ON VALUE=0`

**Comment by xblax** [comment]
Thanks for the suggestion. But I'm not sure we need this by default in the mod because the same can be achieved directly via the Klipper built-in command or from Mainsail/Fluidd/Klipper-Screen UI.

**Comment by cmenard** [comment]
No worries.  I mainly did it to have a simple one button click in Mainsail from my phone.
